templates/admin/org/list.tmpl

```diff
@@ -15,7 +15,6 @@
 					<table class="ui very basic striped table">
 						<thead>
 							<tr>
-								<th>ID</th>
```

Comment:
Please keep that, this is an admin view and I would print there as much information as possible.